addons/stock/report/report_stock_reception.py

```diff
@@ -66,6 +66,13 @@ def _get_report_values(self, docids, data=None):
                         # if immediate transfer is not Done and quantity_done hasn't been edited, then move.product_qty will incorrectly = 1 (due to default)
                         quantity_to_assign = move.product_uom._compute_quantity(move.quantity, move.product_id.uom_id, rounding_method='HALF-UP')
                     product_to_qty_to_assign[move.product_id].append((quantity_to_assign - qty_already_assigned, move))
+            elif move_lines_with_quantity := move.move_line_ids.filtered('quantity_product_uom'):
```

Comment:
in the case that "move.product_qty" is 10 and "qty_already_assigned" is 10, should we do this? The commit message seems to speak about the case of product_qty that is 0.

Reply:
good point, I don't think so